packages/compose/src/hooks/use-focus-on-mount/index.js

```diff
@@ -27,8 +27,16 @@ import { focus } from '@wordpress/dom';
  */
 export default function useFocusOnMount( focusOnMount ) {
 	const didMount = useRef( false );
+	const focusOnMountRef = useRef( focusOnMount );
+	useEffect( () => {
+		focusOnMountRef.current = focusOnMount;
+	}, [ focusOnMount ] );
```

Comment:
Why do we need to update `focusOnMount` after mount if it's only used on mount?

Reply:
It's not used only on mount but on mount of a "ref" which can happen multiple times.